hydra-cluster/test/Test/EndToEndSpec.hs

```diff
@@ -349,6 +354,43 @@ spec = around showLogsOnFailure $ do
                   waitFor tracer 10 [n1] $
                     output "HeadIsOpen" ["utxo" .= Object mempty, "headId" .= headIdAliceOnly]
 
+    describe "fetches correct shelley-genesis values" $
+      it "check shelley-genesis values" $ \tracer ->
+        failAfter 60 $
+          withClusterTempDir "two-heads-cant-abort" $ \tmpDir -> do
+            withCardanoNodeDevnet (contramap FromCardanoNode tracer) tmpDir $ \RunningNode{nodeSocket, networkId} -> do
+              shelleyBS <- readFileBS ("config" </> "devnet" </> "genesis-shelley.json")
+              writeFileBS (tmpDir </> "genesis-shelley.json") shelleyBS
+              loadedShelleyGenesis <- toShelleyGenesis =<< queryGenesisParameters networkId nodeSocket QueryTip
+              expectedShelleyGenesis <- readJsonFileThrow shelleyGenesisFromJson (tmpDir </> "genesis-shelley.json")
```

Comment:
Can't we just check them for equality? i.e. ``loadedShelleyGenesis `shouldBe` expectedShelleyGenesis``?